composer.json

```diff
@@ -27,12 +27,12 @@
     },
     "autoload": {
         "psr-4": {
-            "": "lib/"
+            "Asm89\\Twig\\": "lib/Asm89/Twig/"
```

Comment:
```suggestion
            "Asm89\\": "lib/Asm89/"
```

Reply:
The idea was to add `Twig` to the namespace because this is a Twig-specific repository and there are other libraries of asm89 under the same root namespace `\Asm89`.